DrawerKit/DrawerKit/Public API/Notifications.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+public class NotificationToken {
+    public let token: NSObjectProtocol
+    public let center: NotificationCenter
+
+    init(token: NSObjectProtocol, center: NotificationCenter) {
+        self.token = token
+        self.center = center
+    }
+    
+    deinit { center.removeObserver(token) }
+}
+
+
+public protocol NotificationEnum {
+    var name: Notification.Name { get }
+}
+
+
+extension NotificationCenter {
+    private static let _$key$_ = "_$key$_"
```

Comment:
`_$key$_` any reason to be this format? Couldn't be just `private static let key = "key"`? 
I looked for it and didn't find anything.

Reply:
See https://github.com/Babylonpartners/DrawerKit/pull/36#discussion_r149622350.

Reply:
@daniloaliberti Fixed in https://github.com/Babylonpartners/DrawerKit/pull/36/commits/dedc463d71bee9babf44a73759bebe4f063b4239.